content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +34,193 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs will be removed in cert-manager `v1.6.0`.
-If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
-See cert-manager [docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) for more detailed upgrade instructions.
+These APIs will be removed in cert-manager 1.6.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you are upgrading cert-manager on a cluster which has previously had
+cert-manager < `v1.0.0`, you **must** take steps to upgrade all cert-manager
+custom resources which were created by you or by cert-manager < `v1.0.0`.
+
+Such custom resources may still be stored in `etcd` as `v1alpha2` resources and
+when that version of the API is removed in cert-manager 1.6, they will become
+unreadable.
+
+To work around this, you need to force them to be converted by the cert-manager
+conversion webhook. And the simplest way to do that is to perform a no-change
+update on all cert-manager custom resources.
+
+This is explained in more detail in the [Upgrading existing cert-manager
+resources][upgrade-resources] page.
```

Comment:
```suggestion
⛔️  If you are upgrading cert-manager on a cluster which has previously had
cert-manager < `v1.0.0`, you will need to ensure that all cert-manager custom resources
are stored in etcd at `v1` version and that cert-manger CRDs do not reference the deprecated APIs
**by the time you upgrade to `v1.6`**.

This is explained in more detail in the [Upgrading existing cert-manager
resources][upgrade-resources] page.
```

Reply:
Let me know if this makes sense ^. I think it might make sense to give this a bit less urgency- users only need to do this by the time they upgrade to v1.6 not now, so it might be confusing.